pandas/tools/merge.py

```diff
@@ -280,6 +280,8 @@ def merge_asof(left, right, on=None,
     search selects the row in the right DataFrame whose 'on' key is closest
     in absolute distance to the left's key.
 
+    .. versionadded:: "forward" and "nearest" added in 0.20.0
```

Comment:
I would probably write it in the text that default 'direction' for < 0.20.0 is 'backward', while new options now exist in 0.20.0